app/templates/main/map.html

```diff
@@ -0,0 +1,72 @@
+<html>
+<head>
+<script src="http://maps.googleapis.com/maps/api/js"></script>
+<script>
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
```

Comment:
We should try to use jQuery for getting text from a text box. In general, jQuery abstracts away a lot of cross-browser incompatibility weirdness. While this code will almost certainly work, I think using jQuery is a better practice.